Resources/Prototypes/_White/Catalog/uplink_catalog.yml

```
@@ -52,6 +52,16 @@
       - NukeOpsUplink
   saleLimit: 1
 
+- type: listing
+  id: UplinkImplanterSyndi
+  name: uplink-implanter-name
+  description: uplink-implanter-desc
+  productEntity: ImplanterSyndi
+  cost:
+    Telecrystal: 1
+  categories: 
```

yamllint

[error] trailing spaces

(trailing-spaces)